test/test_autograd.py

```diff
@@ -3871,14 +3874,14 @@ def backward(ctx, grad_out):
                 return NonDetFunc.apply(grad_out, ctx._jitter) * (1 + torch.rand_like(grad_out) * ctx._jitter), None
 
         inp = torch.randn(5, 5, requires_grad=True)
-        gradcheck(lambda x: NonDetFunc.apply(x, 0.0), inp)
+        gradcheck(lambda x: NonDetFunc.apply(x, 0.0), inp, check_batched_grad=False)
```

Comment:
Huh, didn't know this works.

Reply:
This works in what sense? That we have a proper test for non-determinism in the backward?